ResearchKit/ActiveTasks/ORKWaitStepViewController.m

```diff
@@ -0,0 +1,218 @@
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#import "ORKWaitStepViewController.h"
+#import "ORKWaitStep.h"
+#import "ORKCustomStepView_Internal.h"
+#import "ORKActiveStepViewController_internal.h"
+#import "ORKVerticalContainerView.h"
+#import "ORKStepViewController_Internal.h"
+#import "ORKResult.h"
+#import "ORKLabel.h"
+#import "ORKSubheadlineLabel.h"
+#import "ORKHelpers.h"
+#import "ORKAccessibility.h"
+#import "ORKActiveStepView.h"
+
+@interface ORKWaitView: ORKActiveStepCustomView
+
+- (instancetype)initWithIndicatorMask:(ORKProgressIndicatorMask)mask heading:(NSString *)heading;
+
+@property (nonatomic, strong) ORKSubheadlineLabel *textLabel;
+@property (nonatomic, strong) UIProgressView *progressView;
+@property (nonatomic, strong) UIActivityIndicatorView *activityIndicatorView;
+@property (nonatomic, assign) ORKProgressIndicatorMask indictatorMask;
+
+@end
+
+@implementation ORKWaitView
+
+- (instancetype)initWithIndicatorMask:(ORKProgressIndicatorMask)mask heading:(NSString *)heading {
+    self = [super init];
+    if (self) {
+        _indictatorMask = mask;
+        self.translatesAutoresizingMaskIntoConstraints = false;
+        
+        _textLabel = [ORKSubheadlineLabel new];
+        _textLabel.textAlignment = NSTextAlignmentCenter;
+        _textLabel.translatesAutoresizingMaskIntoConstraints = NO;
+        _textLabel.text =  heading ? heading : ORKLocalizedString(@"WAIT_LABEL", nil);
+        [self addSubview:_textLabel];
+        
+        switch (_indictatorMask) {
+            case ORKProgressIndicatorMaskProgressBar:
+                _progressView = [UIProgressView new];
+                _progressView.translatesAutoresizingMaskIntoConstraints = NO;
+                _progressView.progressTintColor = [self tintColor];
+                [self addSubview:_progressView];
+                break;
+            case ORKProgressIndicatorMaskIndeterminate:
+                _activityIndicatorView = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+                _activityIndicatorView.translatesAutoresizingMaskIntoConstraints = NO;
+                _activityIndicatorView.tintColor = [self tintColor];
+                [self addSubview:_activityIndicatorView];
+                [_activityIndicatorView startAnimating];
+                break;
+            case ORKProgressIndicatorMaskNone:
+            default:
+                break;
+        }
+        
+        [self setupConstraints];
```

Comment:
refactor to `setUpConstraints` to be consistent with the latest constraint merge.